tests/e2e/run-e2e.ps1

```
@@ -237,18 +255,25 @@ try {
     # container really is not running, that is the container failing and this line says so,
     # and only once wslc has been made to agree does a disagreeing `wip ps` mean wip is wrong.
+    #
+    # It is the JSON probe that is asserted, not the table: State there is the field wip's own
+    # status and create-vs-start decisions parse, so this assertion covers what wip reads. The
+    # table's STATUS column is prose for a person -- 2.9.9 wrote "running Less than a second
+    # ago" and 2.9.10 writes docker's "Up Less than a second", which names no state at all --
+    # and a lifecycle assertion has no business resting on which phrasing is current.
+    Write-Step "wslc's own view of the container"
+    $probe = Invoke-Wslc @('list', '--all', '--filter', "name=$($script:Container)", '--format', 'json')
+    Assert-Exit $probe 0 "wslc list --format json after wip up"
+    Assert-Match $probe $script:ContainerPattern "wslc list --format json after wip up"
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Assert-Match' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)